### ISSUE TITLE
Allow the cropped file to be automatically written to a file

### ISSUE THREAD

**Issue by vanniktech** [tamimattafi/krop#28]
It would be really nice if the `ImageCropper.cropSrc` function were to be extended with a possibility to write the cropped image to the file system. This is kind of the counterpart of https://github.com/tamimattafi/krop/pull/24 

Regarding writing a file either kotlinxio or okio could be used. Both are multiplatform. Right now the API is kinda cumbersome. I'm getting a `ImageBitmap` back from which I need to read all the pixels manually and write it to a file; or am I overseeing something?

**Comment by tamimattafi** [comment]
@vanniktech I agree that the api is limited to Bitmap right now, and it's kinda frustrating. But I believe after #24, we can add more save destinations, since we now have more data sources.

**Comment by tamimattafi** [comment]
Hello @vanniktech!
In the 0.2.0-alpha02, an extension with FileKit was implemented, does it cover your use-case?
Here's an example on how to use it: https://github.com/tamimattafi/krop/tree/0.2.0?tab=readme-ov-file#filekit

Also, you can use the extension function `bitmap.encodeToByteArray()` to easily convert data to a bytes stream that can be used with kotlinx-io or okio
